docs/docs/quickstart.md

```diff
@@ -14,6 +14,46 @@ $ mkdir quickstart && cd quickstart
 
 </div>
 
+## Create a fleet
+
+Before submitting runs, you need to create a fleet where new instances will be provisioned.
+
+### Define a configuration
+
+Create the following fleet configuration inside your project folder:
+
+<div editor-title="fleet.dstack.yml"> 
+
+```yaml
+type: fleet
+name: default-fleet
+nodes: 0..
```

Comment:
Perhaps, I'd also add `resources` to show that it's possible to limit what GPU types are allowed?